Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap

```diff
@@ -57,6 +59,7 @@ itk_wrap_class("itk::PyVectorContainer")
   endforeach()
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     itk_wrap_template("${ITKM_UC}${ITKM_O${d}}"  "${ITKT_UC},${ITKT_O${d}}")
+      set(ElementType SL)
       set(PyVectorContainerTypes ${ITKM_UC}${ITKM_O${d}})
```

Comment:
@thewtex In other cases `ElementType` is the same as the second template argument, but here `signed long` and `itk::Offset<dim>` are used. Is this related to data representation in `numpy`?

Reply:
Yes, for `OffSet` the element type is `signed long` per:

https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Modules/Core/Common/include/itkIntTypes.h#L94

But, I will change this because, in practice, it changes based on platform, e.g. it could be "signed long long",

https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Modules/Core/Common/include/itkIntTypes.h#L77

So `ITKT_OT` is the right value:
https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Wrapping/WrapBasicTypes.cmake#L189-L204